events/HF_warrior_lodge_flavor_events.txt

```diff
@@ -4608,7 +4608,7 @@ character_event = {
 					30 = {
 						character_event = { id = HF.51006 } #Weapon vanishes
 					}
-					3000 = {
```

Comment:
This is not province ID

Reply:
Fixed in 6d39aee5f58d4d8fbe7faf298e3a9d6e83b76d60.